docs/first_steps.md

```diff
@@ -97,9 +92,11 @@ strong typing.
 Instead, for a coordinate in the `X` dimension, we use `Coordinate<X>` and --as already mentioned--
 for a number of elements in the discretization of `X`, we use `DiscreteVector<DDimX>`.
 
-Once this is done, we define two additional domains.
+Once this is done, we define two additional domains:
++ our zone at the start of the domain that will be mirrored to the ghost.
++ our zone at the end of the domain that will be mirrored to the ghost.
```

Comment:
I guess redundant with the following sentences ?

Reply:
Yes, I could have said it in one sentence

Reply:
I mean the text below explains what the two discrete domains represent right ?

Reply:
Yes